xapian-letor/configure.ac

```diff
@@ -36,7 +36,7 @@ AC_SUBST(LIBRARY_VERSION_INFO)
 
 dnl Where xapian-letor.h, etc go.  In development release append "/xapian-1.3".
 dnl incdir=$includedir
-incdir=$includedir/xapian-1.3
+incdir=$includedir
```

Comment:
Yes, it should be `$includedir/xapian`. -core's configure.ac has the same comment and `incdir` path. That should also be fixed. Will push a change for both of them.